specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/examples/AvailableBalanceByBillingProfile.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "api-version": "2018-11-01-preview",
+    "billingAccountId": "04e9d123-ae03-5621-6c17-e70da00924ee",
```

Comment:
Ideally we use "id" for resource uris, and "name" for actual resource names. I understand subscriptionId does not follow this pattern, but since we're introducing new resources here, let's try to stick to that convention. I.e. use bilingAccountName/billingProfileName in cases like these.